data/json/items/comestibles.json

```diff
@@ -825,6 +825,7 @@
         "volume" : 1,
         "phase" : "liquid",
         "charges" : 5,
+        "flags": [ "EATEN_COLD" ],
```

Comment:
Okay. While I'm at it, I'll make sure all of the `flags` in `comestibles.json` are the last field.